### PR TITLE
Add support for passing in options when intializing a session.

### PR DESCRIPTION
**Pull request by mdillavou**
The initSession has the ability to pass in a keyword list of options [1]. However, the opentok-react doesn't expose this parameter. This is necessary for any customers using enterprise features, like ip whitelisting. This PR allows optionally specifying an options object using the createSession helper or the OTSession class.

[1] https://tokbox.com/developer/sdks/js/reference/OT.html#initSession

### CODE REVIEW

File: src/OTSession.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ export default class OTSession extends Component {
       apiKey: this.props.apiKey,
       sessionId: this.props.sessionId,
       token: this.props.token,
+      options: this.props.options,
```
Comment (msach22) [inline src/OTSession.js]: Since there is no default value, this is causing the tests to fail. Could you please address this?

**Comment by julien-meichelbeck** [comment]
This was committed a year ago, I don't think we'll get an update on this, I can update [my pull-request](https://github.com/opentok/opentok-react/pull/96) since we are in a hurry.

**Comment by julien-meichelbeck** [comment]
@msach22 I updated the PR to introduce the documentation

**Comment by msach22** [comment]
Thank you for the original PR, we've merge another PR with a similar fix: 
https://github.com/opentok/opentok-react/pull/98

We appreciate your contributions and apologies that we were not able to merge this in a timely manner.

**Pull request closed without merging**